test/rocksdb.lua

```diff
@@ -71,6 +71,23 @@ assert(ok == false)
 
 read_only_db:close()
 
+-- open for read only once
+local read_only_db_once = rocksdb.open_for_read_only_once(options, "/tmp/rocksdb.test", false)
```

Comment:
@krijohs what's the false, please name the variable, will be more readable

Reply:
added change